script/build/index.js

```diff
@@ -125,13 +119,27 @@ getElementDefinition
 
 const template = (type) => {
 	switch (type) {
-		case 'bridge':
-			return `if (typeof __require__ !== 'object')
-	return function (r) {
-		factory(window, r)
+		case 'noop':
+			return `
+exports.__SECRET_INTERNALS__ = {
+	mountComponentElement: mountComponentElement,
+	getComponentChildren: getComponentChildren,
+	invokeErrorBoundary: invokeErrorBoundary,
+	getElementDefinition: getElementDefinition
+}
+
+if (typeof __require__ !== 'object')
+	return function (config) {
+		try {
+			factory.call(config, window, config)
+		} catch (e) {
+			/* istanbul ignore next */
+			console.error(err+'\\nSomething went wrong trying to create a custom renderer.')
```

Comment:
Is there a particular reason not to simply let this bubble up?

Reply:
Can't think of any.